apps/komga/1.14.1/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - ./data:/data
       - ./config:/config
-    image: gotson/komga:1.14.1
+    image: gotson/komga:1.15.0
     labels:
       createdBy: "Apps"
 networks:

```

Comment:
The only difference is the upgrade from v1.14.1 to v1.15.0 of GotOn KOMGA, which might include new features, bug fixes, or improvements in performance. No significant changes were detected that would affect functionality but upgrading may bring newer security patches or optimizations. Please ensure compatibility and any additional setup requirements with the latest version before deploying it.